src/terminal/input/terminalInput.hpp

```diff
@@ -17,6 +17,10 @@ Author(s):
 #include "../../types/inc/IInputEvent.hpp"
 #pragma once
 
+// This magic flag is "documented" at https://msdn.microsoft.com/en-us/library/windows/desktop/ms646301(v=vs.85).aspx
+// "If the high-order bit is 1, the key is down; otherwise, it is up."
+static constexpr short KeyPressed{ gsl::narrow_cast<short>(0x8000) };

```

Comment:
is this even used by TerminalInput now? If not, remove it. It's not a part of TerminalInput's public API.

Reply:
It's used in HwndTerminal. But that's it. So I'll just move it over to HwndTerminal then.